packages/urql/updates/index.js

```diff
@@ -1,90 +1,129 @@
-import createComment from './createComment'
+import { get } from 'lodash/fp'
+import meQuery from '@hylo/graphql/queries/meQuery'
 import meCheckAuthQuery from '@hylo/graphql/queries/meCheckAuthQuery'
+import makeAppendToPaginatedSetResolver from './makeAppendToPaginatedSetResolver'
 
 export default {
   Mutation: {
-    addProposalVote: (result, args, cache, info) => { 
+    addProposalVote: (result, args, cache, info) => {
+      // TODO: URQL - This will result in Me being fully re-queried with every new skilled added, which may be fine...
+      // but probably can apply makeAppendToPaginatedSetResolver here as it is a QuerySet
       cache.invalidate({ __typename: 'Post', id: args.postId })
     },
+
     addSkill: (result, args, cache, info) => {
+      // TODO: URQL - This will result in Me being fully re-queried with every new skilled added, which may be fine...
```

Comment:
Ok, I see this one here, maybe the above comment still applies in a related way?

Reply:
Yeah...